db/dummy_package_detail.yml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
can you follow the convention in the folder to have 1 file per model?

computers.yml
electricals.yml
computer_accessories.yml etc etc

makes it easier to add more in the future

Reply:
sure, I'll do that @steveyken .